src/target/tx/radiolink/common/hardware.h

```diff
@@ -53,4 +58,21 @@
 #define _USART_DMA_ISR                dma1_channel4_isr
 
 #include "target/drivers/mcu/stm32/hardware.h"
+
```

Comment:
I recommend have the stm32/hardware.h at the end.  This is where we currently set default for undefined things